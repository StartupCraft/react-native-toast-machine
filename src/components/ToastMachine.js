import React, { Component } from 'react'
import { Text } from 'react-native'

class ToastMachine extends Component {
  render () {
    return <Text>ToastMachine</Text>
  }
}

export default ToastMachine
