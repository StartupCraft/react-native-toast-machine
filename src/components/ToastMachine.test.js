import React from 'react'
import { shallow } from 'enzyme'

import ToastMachine from './ToastMachine'

describe('<ToastMachine>', () => {
  const toastMachine = shallow(<ToastMachine />)

  it('renders properly', () => {
    expect(toastMachine).toMatchSnapshot()
  })
})
