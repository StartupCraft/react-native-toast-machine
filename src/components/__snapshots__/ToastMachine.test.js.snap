// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ToastMachine> renders properly 1`] = `
<Text>
  ToastMachine
</Text>
`;
